### ISSUE TITLE
Chef 12.7.2 && omnitrucker.rb

### ISSUE THREAD

**Issue by destitutus** [hw-cookbooks/omnibus_updater#95]
After update to chef `12.7.2` got this error

```bash
Recipe: custom_cb::default
  * chef_gem[chef-sugar] action upgrade (up to date)

  ================================================================================
  Recipe Compile Error in /var/chef/cache/cookbooks/custom_cb/recipes/default.rb
  ================================================================================

  NameError
  ---------
  uninitialized constant Chef::REST

  Cookbook Trace:
  ---------------
    /var/chef/cache/cookbooks/omnibus_updater/libraries/omnitrucker.rb:84:in `url'
    /var/chef/cache/cookbooks/omnibus_updater/recipes/downloader.rb:27:in `from_file'
    /var/chef/cache/cookbooks/omnibus_updater/recipes/default.rb:25:in `from_file'
    /var/chef/cache/cookbooks/custom_cb/recipes/_packages.rb:9:in `from_file'
    /var/chef/cache/cookbooks/custom_cb/recipes/default.rb:23:in `from_file'
    /var/chef/cache/cookbooks/custom_cb/recipes/default.rb:12:in `from_file'

  Relevant File Content:
  ----------------------
  /var/chef/cache/cookbooks/omnibus_updater/libraries/omnitrucker.rb:

   77:        if(url_or_node.is_a?(Chef::Node))
   78:          url = build_url(url_or_node)
   79:          node = url_or_node
   80:        else
   81:          url = url_or_node
   82:          raise "Node instance is required for Omnitruck.url!" unless node
   83:        end
   84>>       request = Chef::REST::RESTRequest.new(:head, URI.parse(url), nil)
   85:        result = request.call
   86:        if(result.kind_of?(Net::HTTPRedirection))
   87:          result['location']
   88:        end
   89:      end
   90:
   91:    end
   92:  end
   93:
```

All was ok on Chef Client, version `12.6.0`

**Comment by luckymike** [comment]
See discussion here: https://github.com/hw-cookbooks/omnibus_updater/pull/94

**Comment by luckymike** [comment]
I would encourage you to also raise this issue with Chef (possibly the [Chef Board of Governance](https://github.com/chef/chef/blob/master/CBGB.md)), as it seems it was introduced with the release of 12.7, which should not have broken existing functionality (from the Contributing doc):
>* X is a major release, which may not be fully compatible with prior major releases
>* Y is a minor release, which adds both new features and bug fixes
>* Z is a patch release, which adds just bug fixes

We're looking to address this in this cookbook asap; however, at this time the cookbook may only be compatible with `chef =< 12.6`.

**Comment by Stromweld** [comment]
with the pull request merged can we get this version with the fix added to the chef supermarket?

**Comment by slyness** [comment]
@Stromweld #94 was merged and released in omnibus_updater version `1.0.6` .. it is available on the supermarket.
